scanner/engine/master.cpp

```diff
@@ -453,6 +455,34 @@ class MasterImpl final : public proto::Master::Service {
     return grpc::Status::OK;
   }
 
+  grpc::Status HasOp(grpc::ServerContext * context, const proto::HasOpArgs *hasOpArg,
+                    proto::Result *result) {
+    OpRegistry *registry = get_op_registry();
+    if (registry->has_op(hasOpArg->op_name())) {
```

Comment:
```
bool has_op = registry->has_op(hasOpArg->op_name());
result->set_success(has_op);
```

Also conventionally we use Python variable naming convention, e.g. `has_op_arg` instead of `hasOpArg`.